src/Microsoft.Azure.SignalR.Common/Auth/MicrosoftEntra/MicrosoftEntraAccessKey.cs

```diff
@@ -78,22 +78,17 @@ private set
 
     public byte[] KeyBytes => _keyBytes ?? throw new ArgumentNullException(nameof(KeyBytes));
 
-    public Uri Endpoint { get; }
-
     internal Exception? LastException { get; private set; }
 
     internal string GetAccessKeyUrl { get; }
 
     internal TimeSpan GetAccessKeyRetryInterval { get; set; } = TimeSpan.FromSeconds(3);
 
-    public MicrosoftEntraAccessKey(Uri endpoint,
+    public MicrosoftEntraAccessKey(Uri serverEndpoint,
```

Comment:
as the naming convention we used in ServiceEndpoint, this should be the serviceEndpoint?